sanic/server.py

```diff
@@ -298,6 +352,13 @@ def serve(host, port, request_handler, error_handler, before_start=None,
     :param protocol: subclass of asyncio protocol class
     :return: Nothing
     """
+
+    if http2:
+        if not ssl:
```

Comment:
Isn't SSL optional in HTTP2 even though most of the browser doesn't do http2 without ssl?

Reply:
for upstreams and micro-services usages it is better to support http2 without ssl too. As it happened in grpc.